Dockerfile

```diff
@@ -10,43 +10,14 @@ RUN cd /root/ \
  && apt-get update \
  && apt-get install -y python python3 cmake build-essential openjdk-9-jre-headless \
  && /root/emsdk/emsdk update-tags \
- && echo "test latest" \
+ && echo "test the standard workflow a user would do" \
  && /root/emsdk/emsdk install latest \
  && /root/emsdk/emsdk activate latest \
- && python -c "import os ; assert 'fastcomp' in open(os.path.expanduser('~/.emscripten')).read()" \
- && python -c "import os ; assert 'upstream' not in open(os.path.expanduser('~/.emscripten')).read()" \
  && source /root/emsdk/emsdk_env.sh --build=Release \
  && emcc hello_world.cpp \
  && emcc hello_world.cpp -s WASM=0 \
  && emcc --clear-cache \
```

Comment:
Why not put everything above other that the apt-get commands in the test.py script too?

Reply:
I feel safer knowing we run the most basic commands like a user would - in the shell. Python is a layer on top, and maybe the path works differently, or escaping, or the environment, or something else... I changed the comment to reflect that.

Reply:
When I suggest this change it wasn't so much about converting to python but moving into a separate script that can be run outside the Dockfile (not least to avoid the horrible command continuation syntax used in Dockerfiles,).  

So just moving to `test.sh` would be fine with me.  But either way this is an improvement.

Reply:
Oh, I see. Yeah, that would be an additional improvement, I agree.